app/frontend/package.json

```diff
@@ -95,12 +94,15 @@
     "@types/react-test-renderer": "^16.9.3",
     "@types/redux-persist": "^4.3.1",
     "@types/webpack-env": "^1.15.3",
+    "babel-loader": "8.1.0",
     "coveralls": "^3.1.0",
     "eslint-config-prettier": "^6.12.0",
+    "jest-junit": "^12.0.0",
     "lint-staged": "^10.5.1",
     "prettier": "^2.1.2",
     "react-is": "^17.0.1",
-    "react-test-renderer": "^16.13.1"
+    "react-test-renderer": "^16.13.1",
+    "ts-jest": "^26.4.4"
   },
   "jest": {
     "collectCoverageFrom": [
```

Comment:
```json
"setupFilesAfterEnv": ["<rootDir>/src/setupTests.js"]
```

I think you'll probably need this for the tests to run correctly if you are using `jest` directly as opposed to via `react-scripts`. I assume this is to allow you to add custom `reporters`?